src/otx/v2/api/entities/utils/data_utils.py

```diff
@@ -134,8 +134,8 @@ def get_cls_img_indices(labels: List[LabelEntity], dataset: DatasetEntity) -> Di
     """
     img_indices: Dict[str, List[int]] = {label.name: [] for label in labels}
     for i, item in enumerate(dataset):
-        item_labels = item.annotation_scene.get_labels()
-        for i_l in item_labels:
+        # item_labels = item.annotation_scene.get_labels()
+        for i_l in item.annotations:
```

Comment:
How about label instead of i_l?
```suggestion
        for label in item.annotations:
```

Reply:
Thank you, fixed at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.